cadence/tests/test_helpers.cdc

```diff
@@ -78,6 +78,12 @@ access(all) let bridgedTokenCodeChunks = [
f726167652064756520746f206e75616e63657320696e2074686520436164656e63652074657374696e67206672616d65776f726b2e205479706963616c6c792c0a202020202f2f2f2074686520636f6e747261637420776f756c642062652075706461746564207769746820612068617264636f6465642076616c7565206f6e6365207468652045524337323120636f6e74726163742061646472657373206973206b6e6f776e2c2062757420746869732063616e6e6f740a202020202f2f2f2062652064657465726d696e6564207072696f7220746f206465706c6f796d656e7420696e207468652074657374696e6720656e7669726f6e6d656e742e20496e73746561642c207468652045564d206164647265737320697320636f6e6669677572656420696e206163636f756e740a202020202f2f2f2073746f72616765206f6e63652074686520746573742045524337323120636f6e747261637420686173206265656e206465706c6f7965642e0a202020206163636573732873656c66290a20202020766965772066756e2067657445564d436f6e74726163744164647265737328293a2045564d2e45564d41646472657373207b0a202020202020202072657475726e2073656c662e6163636f756e742e73746f726167652e636f70793c45564d2e45564d416464726573733e2866726f6d3a202f73746f726167652f657263373231436f6e747261637441646472657373290a2020202020202020202020203f3f2070616e69632822436f72726573706f6e64696e672045524337323120636f6e7472616374206164647265737320686173206e6f74206265656e20636f6e66696775726564206174202f73746f726167652f657263373231436f6e74726163744164647265737322290a202020207d0a0a20202020696e69742829207b0a0a20202020202020202f2f2053657420746865206e616d65642070617468730a202020202020202073656c662e4d696e74657253746f7261676550617468203d202f73746f726167652f636164656e63654578616d706c654e46544d696e7465720a0a20202020202020202f2f20437265617465206120436f6c6c656374696f6e207265736f7572636520616e64207361766520697420746f2073746f726167650a20202020202020206c657420636f6c6c656374696f6e203c2d2063726561746520436f6c6c656374696f6e28290a20202020202020206c65742064656661756c7453746f7261676550617468203d20636f6c6c656374696f6e2e73746f72616765506174680a20202020202020206c65742064656661756c745075626c696350617468203d20636f6c6c656374696f6e2e7075626c6963506174680a202020202020202073656c662e6163636f756e742e73746f726167652e73617665283c2d636f6c6c656374696f6e2c20746f3a2064656661756c7453746f7261676550617468290a0a20202020202020202f2f206372656174652061207075626c6963206361706162696c69747920666f722074686520636f6c6c656374696f6e0a20202020202020206c657420636f6c6c656374696f6e436170203d2073656c662e6163636f756e742e6361706162696c69746965732e73746f726167652e69737375653c264578616d706c654e46542e436f6c6c656374696f6e3e2864656661756c7453746f7261676550617468290a202020202020202073656c662e6163636f756e742e6361706162696c69746965732e7075626c69736828636f6c6c656374696f6e4361702c2061743a2064656661756c745075626c696350617468290a0a20202020202020202f2f204372656174652061204d696e746572207265736f7572636520616e64207361766520697420746f2073746f726167650a20202020202020206c6574206d696e746572203c2d20637265617465204e46544d696e74657228290a202020202020202073656c662e6163636f756e742e73746f726167652e73617665283c2d6d696e7465722c20746f3a2073656c662e4d696e74657253746f7261676550617468290a202020207d0a7d0a20"
```

Comment:
Included for test purposes. This code allows us to update ExampleNFT as cross-VM and test the case where an NFT is onboarded to the bridge and later updates its EVM association to a project-defined ERC721. Unfortunately, the test suite doesn't have a nicer way of updating contracts in the test environment.